specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -1192,6 +1275,11 @@
           "type": "string",
           "format": "duration",
           "description": "ISO8601 timespan specifying the minimum number of days the environment's events will be available for query."
+        },
+        "storageLimitExceededBehavior": {
+          "type": "string",
+          "description": "The behavior the Time Series Insights service should take when the environment's capacity has been exceeded. If \"PauseIngress\" is specified, new events will not be read from the event source. If \"PurgeOldData\" is specified, new events will continue to be read and old events will be deleted from the environment. The default behavior is PurgeOldData.",
+          "enum": [ "PurgeOldData", "PauseIngress" ]
```

Comment:
note that without `x-ms-enum`, this will not be modeled as an enum in generated SDKs - please add `x-ms-enum` (see other enums in this spec)